### PR TITLE
Use mozlog format in production

### PR DESCRIPTION
**Pull request by mythmon**
Fixes bug 1273331

@Osmose r?

### CODE REVIEW

File: normandy/settings.py [hunk 1]
```diff
@@ -148,6 +148,40 @@ def MIDDLEWARE_CLASSES(self):
...
+            'version': 1,
+            'disable_existing_loggers': True,
+            'formatters': {
+                'json': {
+                    '()': 'mozilla_cloud_services_logger.formatters.JsonLogFormatter',
+                    'logger_name': 'normandy',
+                },
+                'development': {
+                    'format': '%(levelname)s %(asctime)s %(name)s %(message)s',
+                }, },
```
Comment (Osmose) [inline normandy/settings.py]: Put that `}` on another line please.

File: normandy/settings.py [hunk 2]
```diff
@@ -148,6 +148,40 @@ def MIDDLEWARE_CLASSES(self):
...
+            'handlers': {
+                'console': {
+                    'level': 'DEBUG',
+                    'class': 'logging.StreamHandler',
+                    'formatter': 'json' if self.LOGGING_USE_JSON else 'development',
+                },
+            },
+            'root': {
+                'handlers': ['console'],
```
Comment (Osmose) [inline normandy/settings.py]: Maybe make the setting `LOGGING_HANDLERS` and use it here and below? Would allow for a bit more flexibility.

**Comment by Osmose** [comment]
r+wc, nice work!

File: normandy/settings.py (hunk 2 above)
Comment (mythmon) [inline normandy/settings.py]: I don't think this is something we need to be flexible right now. It is easy enough to add later if we need

**Pull request merged**